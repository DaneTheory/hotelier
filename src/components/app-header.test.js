import React from 'react';
import AppHeader from './app-header';
import renderer from 'react-test-renderer';

describe('AppHeader', () => {
  it('matches the snapshot', () => {
    const rendered = renderer.create(<AppHeader/>).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
