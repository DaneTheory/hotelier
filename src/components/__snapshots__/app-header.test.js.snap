// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader matches the snapshot 1`] = `
<header
  className="toolbar toolbar-header"
>
  <h1
    className="title"
  >
    Hotelier
  </h1>
</header>
`;
